src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/IHtmlGenerator.cs

```diff
@@ -157,6 +157,7 @@ public interface IHtmlGenerator
             ModelExplorer modelExplorer,
             string expression,
             object value,
+            string format,
```

Comment:
Nope, it appears `GenerateInput` is not a member of `IHtmlGenerator`. It's an `protected` member of `DefaultHtmlGenerator`.